### DISCUSSION TITLE
ASP.NET running on local ip, not localhost

### DISCUSSION THREAD

**Discussion by tapioca-design** [CommunityToolkit/dotnet#387] [General]
Hi everyone !

On a ASP .NET project, as I needed to see my local work on my phones, I wanted to switch from "localhost" to local ip.

I changed applicationhost.config :

after the existing localhost nodes, adding :
<binding protocol="http" bindingInformation="\*:64645:192.168.XXXX" />
<binding protocol="https" bindingInformation="\*:44314:192.168.XXXX" />

But nothing works, when building and running the project applicationhost.config is overwritten, those two lines disappear and back to initial state, indeed runs localhost:44314

I use JetBrain IDE

If you have any clue, merci !!!

**Comment by hawkerm** [comment]
@tapioca-design looks like your formatting for your code blocks is off or something and not displaying. In either case, this is a discussion forum for the .NET Community Toolkit library and not for general .NET questions. Probably best to join the [C# Discord server](https://discord.gg/csharp) and ask in one of their channels more geared to ASP/Web development.
